frontend/src/store/slice/Standalone/StandaloneSlice.ts

```diff
@@ -0,0 +1,29 @@
+import { createSlice } from "@reduxjs/toolkit"
+
+import { getModeStandalone } from "store/slice/Standalone/StandaloneActions"
+
+const initialState = {
+  mode: false,
```

Comment:
Is it possible to show loading window until it can get mode api result?

For example, when the FE has been started but the BE has not yet been started on application launch.

Reply:
@ReiHashimoto i updated frontend/.env.example and add loading, pls check it again

Reply:
@sanglevinh 
.env.example is OK, thank you.

Apologies, my order for loading was unclear.
mode on initialState should be undefined (maybe loading to be true).
If the mode undefined, app shows only loading icon and retry calling api (with exponential back-off) until it can get mode from BE.

Reply:
@ReiHashimoto i fixed it, pls check it again

Reply:
@sanglevinh 
I made PR about this.
- #191 

I would appreciate your feedback.

Reply:
@ReiHashimoto I think if we do it like this, when the server fails, it will take a very long time to make api calls

Reply:
@sanglevinh 
Thank you.
So, removing exponential back off, just retry in some seconds interval would make sense?